bounties/npm/count-git-tags/1/README.md

```diff
@@ -0,0 +1,27 @@
+## Details:
+I would like to report a `RCE` issue in the `count-git-tags` module.
+It allows to execute `arbitrary commands remotely inside the victim's PC`
+
+## Module
+* **module name:** `count-git-tags`
+* **version:** `1.0.0`
+* **npm page:** `https://www.npmjs.com/package/count-git-tags`
+
+## Module Description
+> A tool to count the number of tags of a git repository
+
```

Comment:
These details are unnecessary for the disclosure

Reply:
Removed in my `master` branch